app/resources/v1/survey.js

```diff
@@ -0,0 +1,45 @@
+const Sequelize = require('sequelize')
+const { Street, Vote } = require('../../db/models')
+const logger = require('../../../lib/logger.js')()
+
+exports.get = async function (req, res) {
+  let ballot
+  try {
+    ballot = await Vote.findAll({
+      where: {
+        voterId: {
+          [Sequelize.Op.is]: null
+        }
+      },
+      order: Sequelize.literal('random()'),
+      limit: 1
+    })
+  } catch (error) {
+    logger.error(error)
+    res.status(500).json({ status: 500, msg: 'Error fetching ballot.' })
```

Comment:
In the staging review app that I've created, I get an error when I hit this URL. I think it's because the review app doesn't have any streets to select. Perhaps in this case we should show a different error message letting us know that there are no streets to retrieve.